src/appdev/ApplicationRouter.js

```diff
@@ -102,7 +102,12 @@ class ApplicationRouter<T> {
     return async (req: Request, res: Response, next: NextFunction) => {
       try {
         const content = await this.content(req);
-        LogUtils.log({ path: this.getPath(), query: req.query });
+        LogUtils.log({
+          path: this.getPath(),
+          query: req.query,
+          serverEpochTime: Date.now(),
+          serverFormattedTime: AppDevUtilities.getESTDateString(),
```

Comment:
this helper function should probably be just defined here, unless other files are going to use it in the future! Also `AppDevUtilities` should be `ApplicationUtils`, we want to keep the files/imports consistent

Reply:
you can just use the `_` convention that Pollo uses, for private functions in a class!

Reply:
utils should be helper methods though right? applicationutils is just a file that holds helper functions for applicationrouter

Reply:
also can we add javadoc, and change the name to `getESTDateStringNow`

Reply:
there's only one other function that's being used in `ApplicationUtils`, can we just deprecate the file completely and move these two functions here? 

cc: @meganle thoughts

Reply:
well the other function in `ApplicationUtils` should stay since that verifies all of the urls in all of the routers. i think it's fine to add helper methods to that file. i know pollo uses it for misc helper functions so it should be fine there